assets/section-main-product.css

```
@@ -1322,6 +1323,16 @@ a.product__text {
   margin-bottom: var(--icon-size);
 }
 
+/* SKU block */
+
+p.visibility-hidden:empty {
+  display: block;
+}
+
+p.visibility-hidden:empty::after {
+  content: "#";
+}
```

I think these better to be changed to something more specific for SKU.
```suggestion
product__sku.visibility-hidden:empty {
  display: block;
}

product__sku.visibility-hidden:empty::after {
  content: "#";
}
```